### DISCUSSION TITLE
NGA上帖子底部广告关闭误触问题

### DISCUSSION THREAD

**Discussion by yjg954102** [AIsouler/GKD_subscription#400] [Q&A] [answered]
NGA底部关闭广告规则看不到底部广告时总是会无效反复触发，滑到底部时会点到广告跳转第三方软件，能麻烦大佬看看规则能怎么修改吗？https://github.com/user-attachments/assets/8d0c658a-07eb-4291-b542-94dcd5cdc9bc

[NGA玩家社区_ArticleDetailActivity-1728694772679.zip](https://github.com/user-attachments/files/17348341/NGA._ArticleDetailActivity-1728694772679.zip)

**Comment by AIsouler** [comment, marked as answer]
03be8c4cf456806abd2054f6f5a9db22cde29bb0
